plugins/essif-lab_contactform7/src/Implementation/Hooks.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TNO\ContactForm7\Implementation;
+
+class Hooks
+{
+    private $plugin_name;
+    private $version;
+
+    public function __construct($plugin_name, $version)
+    {
+        $this->plugin_name = $plugin_name;
+        $this->version = $version;
+    }
+
+    public function enqueue_scripts()
+    {
+        wp_enqueue_script( $this->plugin_name, plugin_dir_url( __FILE__ ) . '../js/script.js', array( 'jquery' ), $this->version, false );
```

Comment:
Dit wordt nu volgens mij op elke pagina geladen (behalve admin paginas). Probeer de loads te beperken.

Reply:
Voor zover ik heb kunnen vinden alleen mogelijk met specifieke id's of templates waar wij geen gebruiken van maken. Misschien weet jij hoe dit mogelijk is?